src/renderer/components/chart/pie-chart.scss

```diff
@@ -1,6 +1,7 @@
 .PieChart {
   .chart-container {
     width: 120px;
+    min-height: 150px;
```

Comment:
I noticed an issue. When I turn on the metrics again pie-charts are not displayed (without refresh). I see in devTools that their height is equal to 0. Could  I set `min-height` to fix it? It works for me 
<img width="572" alt="Screen Shot 2021-02-18 at 9 58 05 PM" src="https://user-images.githubusercontent.com/38247153/108402501-0ce47a80-7237-11eb-8e8f-9c782c30fa01.png">